cmd/cmd_consts.go

```diff
@@ -19,3 +19,5 @@ const sortDirection = "sort-direction"
 const maxListSize = "max-list-size"
 
 const tokenEnvVarKey = "token-env-var-key"
+
+const increment = "increment"
```

Comment:
Wanted to double-check the name of the option. Used it because it seems to be the (internal) nomenclature used for the `release` command. 
Iirc Axion uses `incrementer` (or version incrementer). 

A couple of options to consider:
- `--increment` (as is)
- `--incrementer`
- `--version-incrementer`
- `--increment-strategy`
- ??

Any thoughts?

Reply:
I personally am a fan of --increment